packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx

```diff
@@ -18,28 +20,18 @@ describe('<CIDocument />', () => {
   let getAllByRole: BoundFunction<GetAllBy<any[]>>,
     getByTestId: BoundFunction<GetByText>,
     getByText: BoundFunction<GetByText>,
-    getByTitle: BoundFunction<GetByText>;
+    getByTitle: BoundFunction<GetByText>,
+    findByText: BoundFunction<FindByText>;
 
   beforeEach(() => {
     act(() => {
-      ({ getAllByRole, getByTestId, getByText, getByTitle } = render(
+      ({ getAllByRole, getByTestId, getByText, getByTitle, findByText } = render(
         <CIDocument document={contract} overrideDocWidth={400} overrideDocHeight={600} />
       ));
     });
   });
 
-  it('renders without crashing', async () => {
-    jest.setTimeout(10000);
-    await waitForElement(() =>
-      // check for (partial) document text
-      getByText('On 22 December 2008 ART EFFECTS LIMITED and Customer', { exact: false })
-    );
-    // check for file name
-    getByText('Art Effects Koya Creative Base TSA 2008.pdf');
-    // check for a filter name
-    const filters = getByTestId('Filters');
-    globalGetByText(filters, 'Amendments');
-  });
+  afterEach(cleanup);
```

Comment:
This isn't necessary, since it's done automatically in jest ([docs](https://testing-library.com/docs/react-testing-library/api#cleanup)):

> Please note that this is done automatically if the testing framework you're using supports the `afterEach` global (like mocha, Jest, and Jasmine).